osu.Game/Overlays/Chat/ChatLine.cs

```diff
@@ -223,7 +223,10 @@ private void updateMessageContent()
             timestamp.FadeTo(message is LocalEchoMessage ? 0 : 1, 500, Easing.OutQuint);
 
             timestamp.Text = $@"{message.Timestamp.LocalDateTime:HH:mm:ss}";
-            username.Text = $@"{message.Sender.Username}" + (senderHasBackground || message.IsAction ? "" : ":");
+
+            username.Text = message.Sender.Username;
+            username.Truncate(message_padding - 20, ellipsis: true);
+            username.Text = $@"{username.Text}" + (senderHasBackground || message.IsAction ? "" : ":");
```

Comment:
This could just be `username.Text + ...` The interpolation shouldn't be necessary.